.gitignore

```diff
@@ -54,3 +54,28 @@ pom.xml.asc
 /fixtures/*.db-shm
 /fixtures/*.db-wal
 /query-parser/out/
+## Build generated
+build/
```

Comment:
This isn't right -- `build/version.rs` exists in the tree.  Is this written by XCode?  Can we make it not be written by XCode?  Or should we move our build script?

Reply:
No, this was my mistake. This should be specific for Xcode `build/` directory, not the root one.